cdap-distributions/src/hdinsight/install.sh

```diff
@@ -22,9 +22,9 @@
 # The git branch to clone
 CDAP_BRANCH='release/3.5'
 # Optional tag to checkout - All released versions of this script should set this
-CDAP_TAG='v3.5.0'
+CDAP_TAG=''
 # The CDAP package version passed to Chef
-CDAP_VERSION='3.5.0-1'
+CDAP_VERSION='3.5.1-1'
```

Comment:
Sorry. It needed to be updated to `3.5.1`, not to `3.6.0`. I thought that Ali was asking if that change was correct/necessary. It was correct as it was. It *must* match a released version of CDAP.